src/NRediSearch/Aggregation/AggregationBuilder.cs

```diff
@@ -0,0 +1,157 @@
+﻿// .NET port of https://github.com/RedisLabs/JRediSearch/
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using NRediSearch.Aggregation.Reducers;
+
+namespace NRediSearch.Aggregation
+{
+    public class AggregationBuilder
+    {
+        private readonly List<object> _args = new List<object>();
+
+        public bool IsWithCursor { get; private set; }
+
+        public ReadOnlyCollection<object> Args => new ReadOnlyCollection<object>(_args);
+
+        public string ArgsString => string.Join(" ", _args.ToArray());
+
+        public AggregationBuilder() : this("*")
+        {
+        }
+
+        public AggregationBuilder(string query) => _args.Add(query);
+
+        public AggregationBuilder Load(params string[] fields)
+        {
+            AddCommandArguments(_args, "LOAD", fields);
+
+            return this;
+        }
+
+        public AggregationBuilder Limit(int offset, int count)
+        {
+            var limit = new Limit(offset, count);
+
+            limit.SerializeRedisArgs(_args);
+
+            return this;
+        }
+
+        public AggregationBuilder Limit(int count) => Limit(0, count);
+
+        public AggregationBuilder SortBy(params SortedField[] fields)
+        {
+            _args.Add("SORTBY");
+            _args.Add((fields.Length * 2).ToString());
```

Comment:
invariant culture (note to self: see if this is exposed anywere in a single place)